runtime/kusama/src/weights/pallet_democracy.rs

```diff
@@ -174,9 +174,9 @@ impl<T: frame_system::Config> pallet_democracy::WeightInfo for WeightInfo<T> {
 	// Storage: Democracy ReferendumInfoOf (r:1 w:0)
 	/// The range of component `r` is `[1, 99]`.
 	fn on_initialize_base(r: u32, ) -> Weight {
-		(1_518_000 as Weight)
+		(3_049_000 as Weight)
```

Comment:
weight x2

Reply:
SWC now supports correct worst-case weight analysis by taking the ranges of the parameters into account.  
In this example the base-weight went up a lot, but the weight of component `r` just changed from `3_035_000` to `3_067_000`. Since component `r` has a max value of 99, the worst-case weight is just 1.54% higher.  
Visible [here](https://weights.tasty.limo/compare?old=release-v0.9.26&new=coderobe%2Fweights-26&threshold=1&unit=weight&repo=polkadot&path_pattern=runtime%2F*%2Fsrc%2Fweights%2F*.rs&method=worst&ignore_errors=false#runtime/kusama/src/weights/pallet_democracy.rs.on_initialize_base) (takes a while to load).